tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
No it's not. That looks like IE-specific extensions and definitely isn't ES spec.